apps/ensrainbow/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "commonjs",
+    "lib": ["ES2020"],
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "outDir": "dist",
+    "rootDir": "src"
+  },
```

Comment:
can you make this tsconfig extend from the shared lib tsconfig? if anything specific needs to be changed, you can override it

```json
{
  "extends": "@namehash/shared-configs/tsconfig.lib.json",
  "include": ["src/**/*"],
  "exclude": ["node_modules"]
}
```